lib/membrane_file/sink.ex

```diff
@@ -15,12 +22,27 @@ defmodule Membrane.File.Sink do
   @common_file Membrane.File.CommonFileBehaviour.get_impl()
 
   def_options location: [
-                spec: Path.t(),
-                description: "Path of the output file"
+                spec: Path.t() | :stdout,
+                description: "Path of the output file or :stdout"
               ]
 
   def_input_pad :input, flow_control: :manual, demand_unit: :buffers, accepted_format: _any
 
+  @spec redirect_logs_to_stderr() :: :ok
+  def redirect_logs_to_stderr() do
```

Comment:
A function's documentation would be welcome. I would also think of putting that function in a separate module, (something like `Membrane.File.SupportForStdio`), WDYT?